source/manual/manage-ruby-dependencies.html.md

```diff
@@ -5,29 +5,42 @@ description: How we manage our Ruby dependencies using Dependabot, who can merge
 section: Dependencies
 layout: manual_layout
 parent: "/manual.html"
-last_reviewed_on: 2019-04-17
+last_reviewed_on: 2019-11-04
 review_in: 6 months
 ---
 
 We're [obliged to keep our software current][current].
 
 To help with this, we use a service called [Dependabot][] to perform automated dependency upgrades.
 
-## Add Dependabot to a repo
+## Dependabot for new dependency versions
+
+### Add Dependabot to a repo
 
 1. Give Dependabot [access to the repo][access] (only GitHub org owners can do this)
-2. Go to [Dependabot admin][admin] and click "Add project"
+1. Go to [Dependabot admin][admin] and click "Add project"
 
-## Ask Dependabot to bump dependencies
+### Ask Dependabot to bump dependencies
 
 By default Dependabot will bump dependencies once a day, but you can ask it to bump manually:
 
 Go to [Dependabot admin][admin] and click "Bump now" for your project
 
-## Security
+### Security
 
 There are 2 safeguards to prevent unauthorised code changes. Firstly, Dependabot can only update the repositories that we [explicitly allow on GitHub][access]. This prevents code changes to other repos. Secondly, we've [set up branch protection](/manual/configure-github-repo.html#auto-configuration) for all repos with the `govuk` label. This prevents Dependabot from writing directly to master.
 
+## Dependabot for security vulnerabilities in dependencies
+
+Since GitHub acquired Dependabot, there are two ways that it updates
+dependencies in your project. Security vulnerabilities come through
+the GitHub dependabot channel under the username `dependabot-preview`.
```

Comment:
What's the significance of the `dependabot-preview` username, as this is the one that's used for general Dependabot Pull Requests as well right?

Reply:
Dependabot raises PRs under two different names (one's the "main" one and one's the "GitHub" one). Maybe I got the wrong one for these instructions. Not particularly significant though. I'll remove it.

Reply:
Ok :+1: